vmck/backends/submission.py

```diff
@@ -0,0 +1,36 @@
+from . import qemu
+
+
+class Submission:
+
+    def task_group(self, job, options):
+        submission_task = {
+            'name': 'submission-handler',
+            'driver': 'docker',
+            'config': {
+                'image': 'vmck/vagrant-vmck:submission',
```

Comment:
Use the `vmck/vagrant-vmck-acs` image here (https://github.com/vmck/vagrant-vmck/pull/23#discussion_r322408633) and remove `DOWNLOAD_SCRIPT_URL` from the environment.